externals/sha3/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(SHA3)
```

Comment:
The `configureHook` and `makeHook` from the src subdirectory should be removed

Reply:
Sure, will do.